tests/e2e/specs/shopper/cart-checkout/checkout.test.js

```diff
@@ -36,9 +35,18 @@ if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
 	test.only( 'Skipping Cart & Checkout tests', () => {} );
 }
 
-let companyCheckboxId = null;
 let coupon;
 
+const getCompanyCheckboxLabel = async () => {
+	await page.waitForXPath(
+		`//label[contains(text(), "Company") and contains(@class, "components-toggle-control__label")]`
+	);
+	const [ companyCheckboxLabel ] = await page.$x(
+		`//label[contains(text(), "Company") and contains(@class, "components-toggle-control__label")]`
+	);
```

Comment:
I believe `waitForXPath` returns the Element handle, so you can already assign it there.

Reply:
True: https://pptr.dev/api/puppeteer.page.waitforxpath/. Thank you so much!

Reply:
Addressed in [0afa90b](https://github.com/woocommerce/woocommerce-blocks/pull/7554/commits/0afa90bfef40c5055573869e81be83aa7dda161c)